store/tikv/tikvrpc/tikvrpc.go

```diff
@@ -448,25 +448,25 @@ func CallRPC(ctx context.Context, client tikvpb.TikvClient, req *Request, ch cha
 // TimeoutItem is used to implement grpc stream timeout.
 type TimeoutItem struct {
 	cancel   context.CancelFunc
-	timeout  time.Time
-	complete int32 // atomic variable, 0 means false
+	deadline int64 // A time.Unix value, if time.Now().Unix() > deadline, cancel() would be called.
 }
 
 // Recv overrides the stream client Recv() function.
 func (resp *CopStreamResponse) Recv() (*coprocessor.Response, error) {
-	item := &TimeoutItem{
-		cancel: resp.cancel,
-		// TODO: Make timeout a field in CopStreamResponse.
```

Comment:
Please don't remove the TODO.

Reply:
I'll make a issue and work on it right after this PR